lte/gateway/c/core/oai/tasks/nas5g/src/M5GAuthenticationResponse.cpp

```diff
@@ -26,43 +26,37 @@ AuthenticationResponseMsg::AuthenticationResponseMsg(){};
 AuthenticationResponseMsg::~AuthenticationResponseMsg(){};
 
 // Decode AuthenticationResponse Messsage
-int AuthenticationResponseMsg::DecodeAuthenticationResponseMsg(
-    AuthenticationResponseMsg* auth_response, uint8_t* buffer, uint32_t len) {
+int AuthenticationResponseMsg::DecodeAuthenticationResponseMsg(uint8_t* buffer,
+                                                               uint32_t len) {
```

Comment:
No change in indentation has been made, only the removal of unnecessary field to avoid misuse of method and ease the comprehension of the code.
The indentation is automatically placed in the devcontainer environment.